routes/teams.js

```diff
@@ -25,7 +25,7 @@ router.get("/:slug", async (req, res) => {
   res.status(200).json(team);
 });
 
-router.get("/:id", auth, async (req, res) => {
+router.get("/u/:id", auth, async (req, res) => {
```

Comment:
Typo? Not sure what this change was for. If it's intended, lemme know what it was for.

Reply:
there's already a get('/:slug') route
so a get(/:id) won't work since the backend doesn't know which route you're trying to hit.
therefore I added the u ***as in user*** to separate the routes
- unless you have middleware checking it
- But I didn't feel like writing middleware so I just added a u

lol